airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -1808,6 +1808,8 @@ def __init__(
         gcp_conn_id: str = "google_cloud_default",
         impersonation_chain: Optional[Union[str, Sequence[str]]] = None,
         asynchronous: bool = False,
+        hook: Optional[DataprocHook] = None,
+        job_id: str = '',
```

Comment:
```suggestion

```
No need for this one. The `job_id` should be assigned in `execute` method that is executed before `on_kill`.